prow/cmd/hook/main.go

```diff
@@ -220,5 +220,20 @@ func main() {
 	// Serve plugin help information from /plugin-help.
 	http.Handle("/plugin-help", pluginhelp.NewHelpAgent(pluginAgent, githubClient))
 
-	logrus.Fatal(http.ListenAndServe(":"+strconv.Itoa(o.port), nil))
+	httpServer := &http.Server{Addr: ":" + strconv.Itoa(o.port), Handler: nil}
```

Comment:
as discussed, probably we can drop `Handler: nil`?

Reply:
this doesn't appear to be dropped